packages/qdrant-loader/src/qdrant_loader/core/pipeline/orchestrator.py

```
@@ -127,44 +222,125 @@ async def process_documents(
-            )
-            self.last_pipeline_result = result
+                async for batch in stream_iter:
+                    total_documents += len(batch)
+                    batch_count += 1
 
-            # Update document states for successfully processed documents
-            await self._update_document_states(
-                documents, result.successfully_processed_documents, current_project_id
-            )
+                    if not force and change_detector is not None:
+                        batch = await change_detector.classify_batch(
```

Ruff (0.15.14)

[warning] `for` loop variable `batch` overwritten by assignment target

(PLW2901)